<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OtherOperateWidget</class>
 <widget class="QWidget" name="OtherOperateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>414</width>
    <height>113</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>414</width>
    <height>113</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OtherOperateWidget</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>410</width>
     <height>111</height>
    </rect>
   </property>
   <widget class="OperatorButton" name="otherOperator_07">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>0</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_06">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_03">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>58</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_01">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>58</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_04">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>58</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_02">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>58</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_05">
    <property name="geometry">
     <rect>
      <x>327</x>
      <y>58</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_08">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>0</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="OperatorButton" name="otherOperator_09">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>0</y>
      <width>78</width>
      <height>53</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>78</width>
      <height>53</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>53</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OperatorButton</class>
   <extends>QPushButton</extends>
   <header location="global">OperatorButton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
